styles/pythonanywhere/catppuccin.user.css

```diff
@@ -0,0 +1,1553 @@
+      );
```

Comment:
Assuming it is the same SVG you could just move this logo selector to the line above with `img[src="/static/anywhere/images/PA-logo.svg"]` and remove the duplicate? If so please leave a comment as well saying that `img[src*="/images/PA-logo.svg"]` is a selector from the blog page.

Reply:
the blog selector uses the url `./images/PA-logo.svg` where the main website uses `/static/anywhere/images/PA-logo.svg`. I've updated the selector to better reflect this

Reply:
Right but isn't it the same SVG image? Why not just put both selectors for the same rule that applies the SVG?

Reply:
is there a way to do that even though they are both in their own `@-moz-document`?

Reply:
Everything in the `@-moz-document domain("pythonanywhere.com")` will also apply to the blog.pythonanywhere.com subdomain, the only reason we have a different `@-moz-document` for the blog subdomain is for organization. So if you just add the selector for the image on the blog page to the existing CSS rule for the logo on the other page it should just work.

Reply:
The top level `@-moz-document` is only for `www.-` and `eu.pythonanywhere.com` since users' websites get hosted on `username.pythonanywhere.com`

Reply:
Well then we could just add `domain("blog.pythonanywhere.com")` to the first moz-document thingie?